src/components/CookbookDocItem.tsx

```diff
@@ -51,27 +51,108 @@ function InnerCookbookDocItem({content, tags}: CookbookDocItemProps) {
 
   // Now we're under <DocProvider>, so useDoc() is safe:
   const {metadata, frontMatter, toc, contentTitle} = useDoc();
-  const {
-    title,
-    description,
-    id,
-    unversionedId,
-    tags: metaTags = [],
-    formattedLastUpdatedAt,
-    lastUpdatedAt,
-  } = metadata as typeof metadata & {
-    unversionedId?: string;
-    formattedLastUpdatedAt?: string;
-    lastUpdatedAt?: number | string | null;
-  };
+  const {title, description, id, unversionedId, tags: metaTags = [], formattedLastUpdatedAt, lastUpdatedAt} = metadata as
```

Comment:
Minor nit: Why change the spacing here? It's a little easier to read in the old format.